RacingMod/RacingTools.cs

```diff
@@ -83,5 +84,12 @@ public static void ShowError (Exception e, Type type)
             if (MyAPIGateway.Session?.Player != null)
                 MyAPIGateway.Utilities.ShowNotification($"[ ERROR: {type.FullName}: {e.Message} | Send SpaceEngineers.Log to mod author ]", 16, MyFontEnum.Red);
         }
+
+        public static bool IsPlayerAdmin(IMyPlayer p)
+        {
+            if (p.SteamUserId == 76561198082681546L)
```

Comment:
Development time convenience code. I suggest removing it or adding it to the configuration file. That way it could be configured on your development client (or server), but not hardcoded into the source code.

Reply:
This line is important because it allows me to intervene and fix someone's map live without needing admin. It's unfortunately happened many times